model_analyzer/analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -27,16 +27,16 @@
 from itertools import product
 from subprocess import check_output, CalledProcessError, STDOUT
 
-from model_analyzer.analyzer.perf_analyzer.perf_record import PerfRecord
 from model_analyzer.model_analyzer_exceptions import TritonModelAnalyzerException
+from .perf_latency import PerfLatency
+from .perf_throughput import PerfThroughput
 
 
 class PerfAnalyzer:
     """
     This class provides an interface for running workloads
     with perf_analyzer.
     """
-
     def __init__(self, config):
```

Comment:
Leave the empty line. Vertical spacing is important to readability and clean code.

Reply:
I'm using yapf python fomatting (It's what is used in triton as well). It deletes lines like this.

Reply:
that's inconsistent. The empty line exists in other files.

Reply:
We should change everything to match this format. VSCode supports the yapf autoformatting as well. I will do this to all the files in my PRs. @Tabrizian should do the same.

Reply:
Sure, I will do this too.

Reply:
Can you change the YAPF formatting to add the line. Without it, it looks like the descriptor will go to __init__, not the class. Unless a comment is tied to a specific line of code, there should always be an empty line after a comment.